checks.d/kubernetes.py

```diff
@@ -220,6 +222,8 @@ def _retrieve_metrics(self, url):
     def _retrieve_kube_labels(self):
         return get_kube_labels()
 
+    def _retrieve_pods_list(self, url):
```

Comment:
Nitpicking but the name `_retrieve_pods_list` sounds like it will do the work itself, without needing to pass it a url. The signature should be `_retrieve_pods_list(self)`. See `_retrieve_kube_labels` that follows this style.